Formula/recipes.rb

```diff
@@ -1,17 +1,19 @@
 class Recipes < Formula
   desc "Formula for GNOME recipes"
   homepage "https://wiki.gnome.org/Apps/Recipes"
-  url "https://download.gnome.org/sources/recipes/0.4/recipes-0.4.2.tar.xz"
-  sha256 "9554d4f5d97eb9cd4032de0e4f9cc27a218c32d022dd1917a7e9efbd379c5bc1"
+  url "https://download.gnome.org/sources/gnome-recipes/0.22/gnome-recipes-0.22.0.tar.xz"
+  sha256 "ae4f669b1f1d20f2846ad1c9f6ba6580a15347288ef3d044972c2ba589d9c7b9"
```

Comment:
I think it has happened in the past: rename + alias for the old name
https://github.com/Homebrew/homebrew-core/commit/2c6fd9c6c0e97c68debe2b07593c6e61558f9713